INSTALLATION.md

```diff
@@ -1,29 +1,127 @@
 # Installation
 
-The latest stable version of _taipy-gui_ is available through _pip_:
+There are different ways to install Taipy GUI, depending on how
+you plan to use it.
+
+If your goal is to look into the code, modify and improve it, go straight
+to the [source installation](#installing-for-development) section.
+
+Taipy GUI needs your system to have Python 3.8 or above installed.
+
+## Installing from PyPI
+
+The easiest way to install Taipy GUI is from the
+[Pypi software repository](https://pypi.org/project/taipy-gui/).
+
+Run the command:
+```console
+$ pip install taipy-gui
 ```
-pip install taipy-gui
+
+If you are running in a virtual environment, you will have to
+issue the command:
+```console
+$ pipenv install taipy-gui
 ```
 
-## Development version
+These commands install the `taipy-gui` package in the Python environment
+with all its dependencies.
 
-You can install the development version of _taipy_ with _pip_ and _git_:
-```
-pip install git+https://git@github.com/Avaiga/taipy-gui
+## Installing from GitHub
+
+The development version of Taipy GUI is updated daily with changes from the
+Taipy R&D and external contributors that we praise for their input.
+
+This development version of Taipy GUI can be installed using _pip_ and _git_:
+```console
+$ pip install git+https://git@github.com/Avaiga/taipy-gui
 ```
 
-## Work with the _taipy-gui_ code
+## Installing for development
+
+If you need the source code for Taipy GUI on your system, so you can see
```

Comment:
```suggestion
If you need the source code for Taipy GUI on your system so you can see
```

Reply:
I don't agree with this one.
The part within commas really is an optional part of the sentence...